apps/ios/src/Podfile.lock

```diff
@@ -9,15 +9,15 @@ PODS:
     - React-Core (= 0.62.2)
     - React-jsi (= 0.62.2)
     - ReactCommon/turbomodule/core (= 0.62.2)
-  - FluentUI-React-Native-Avatar (0.4.1):
+  - FluentUI-React-Native-Avatar (0.4.3):
     - MicrosoftFluentUI/AvatarView_mac (~> 0.1.28)
     - MicrosoftFluentUI/Controls_ios (~> 0.1.28)
     - React
-  - FluentUI-React-Native-Button (0.2.1):
+  - FluentUI-React-Native-Button (0.3.1):
     - MicrosoftFluentUI/Button_mac (~> 0.1.28)
     - MicrosoftFluentUI/Controls_ios (~> 0.1.28)
```

Comment:
fyi next fluent publish, this need to change to button_ios and avatar_ios!

Reply:
Yeah, I should probably log a bug to update all the FluentUI Apple dependencies